code/modules/surgery/organs/lavaland.dm

```diff
@@ -0,0 +1,233 @@
+		for(var/mob/living/L in listeners)
+			var/throwtarget = get_edge_target_turf(owner, get_dir(owner, get_step_away(L, owner)))
+			L.throw_at_fast(throwtarget, 3, 1)
+		next_command = world.time + cooldown_damage
+
+	//WALK
+	else if(findtext(command, "walk"))
+		for(var/mob/living/L in listeners)
+			L.m_intent = MOVE_INTENT_WALK
+		next_command = world.time + cooldown_meme
+
+	//RUN
+	else if(findtext(command, "run"))
+		for(var/mob/living/L in listeners)
+			L.m_intent = MOVE_INTENT_RUN
+		next_command = world.time + cooldown_meme
+
+	//HELP INTENT
+	else if(findtext(command, "help"))
+		for(var/mob/living/L in listeners)
+			if(ishuman(L))
+				var/mob/living/carbon/human/H = L
+				H.a_intent_change(INTENT_HELP)
+		next_command = world.time + cooldown_meme
+
+	//DISARM INTENT
+	else if(findtext(command, "disarm"))
+		for(var/mob/living/L in listeners)
+			if(ishuman(L))
+				var/mob/living/carbon/human/H = L
+				H.a_intent_change(INTENT_DISARM)
+		next_command = world.time + cooldown_meme
+
+	//GRAB INTENT
+	else if(findtext(command, "grab"))
+		for(var/mob/living/L in listeners)
+			if(ishuman(L))
+				var/mob/living/carbon/human/H = L
+				H.a_intent_change(INTENT_GRAB)
+		next_command = world.time + cooldown_meme
+
+	//HARM INTENT
+	else if(findtext(command, "harm"))
+		for(var/mob/living/L in listeners)
+			if(ishuman(L))
+				var/mob/living/carbon/human/H = L
+				H.a_intent_change(INTENT_HARM)
+		next_command = world.time + cooldown_meme
+
+	//THROW/CATCH
+	else if(findtext(command, "throw") || findtext(command, "catch"))
+		for(var/mob/living/L in listeners)
+			if(iscarbon(L))
+				var/mob/living/carbon/C = L
+				C.throw_mode_on()
+		next_command = world.time + cooldown_meme
+
+	//FLIP
+	else if(findtext(command, "flip") || findtext(command, "rotate") || findtext(command, "revolve") || findtext(command, "roll"))
+		for(var/mob/living/L in listeners)
+			L.emote("flip")
+		next_command = world.time + cooldown_meme
+
+	//SIT
+	else if(findtext(command, "sit"))
+		for(var/mob/living/L in listeners)
+			for(var/obj/structure/chair/chair in get_turf(L))
+				chair.buckle_mob(L, force=1)
+		next_command = world.time + cooldown_meme
+
+	//STAND UP
+	else if(findtext(command, "stand"))
+		for(var/mob/living/L in listeners)
+			if(L.buckled)
+				L.buckled.unbuckle_mob(L, force = 1)
+		next_command = world.time + cooldown_meme
+
+	//DANCE
+	else if(findtext(command, "dance"))
+		for(var/mob/living/L in listeners)
+			L.emote("dance")
+		next_command = world.time + cooldown_meme
+
+	//JUMP
+	else if(findtext(command, "jump"))
+		for(var/mob/living/L in listeners)
+			L.say("HOW HIGH?!!")
+			L.emote("jump")
+		next_command = world.time + cooldown_meme
+
+	//SALUTE
+	else if(findtext(command, "salute"))
+		for(var/mob/living/L in listeners)
+			L.emote("salute")
+		next_command = world.time + cooldown_meme
+
+	//PLAY DEAD
+	else if(findtext(command, "play dead"))
+		for(var/mob/living/L in listeners)
+			L.emote("deathgasp")
```

Comment:
To be fair, that would require a miner to get to a collosus, kill it, make it back to station AND have the surgery done before the ops have secured the disk/killed everyone.